tests/scripts/common/common_functions.sh

```diff
@@ -277,7 +277,8 @@ function autotune_cleanup() {
 	echo "CMD = ${cmd}"
 	${cmd} >> ${AUTOTUNE_SETUP_LOG} 2>&1
 	# Remove the prometheus setup
-	prometheus_cleanup
+	# Uncomment this line if prometheus setup also needs to be cleaned up
+	# prometheus_cleanup
```

Comment:
Maybe this needs to be a parameter? Especially because this is common code

Reply:
Thanks Dino, I have updated the scripts with an option to cleanup prometheus.